components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -1043,6 +1045,7 @@ export class AppCommandParser {
                     Description: binding.description || '',
                     Hint: binding.hint || '',
                     IconData: binding.icon || '',
+                    type: Constants.Integrations.COMMAND_SUGGESTION_DEFAULT,
```

Comment:
Sorry I meant we should use `type: 'command'` here, since these are command name suggestions, and not command arguments. The only change I was suggesting we remove is in `app_provider`. All other `command` suggestions can stay as `suggestion.command`. What do you think about this? And I apologize about the back-and-forth on this @sinansonmez!